docs/controls/InAppNotification.md

```diff
@@ -101,23 +142,55 @@ private void InAppNotification_OnDismissed(object sender, EventArgs e)
 
 ## Animation
 
-By default, the popup animation of the control is a bottom to top animation. You can update the popup animation using the `RenderTransformOrigin` property of the control. See examples:
+The default animation are set on each Notification Style. 
+You can update the animation using three distinct properties :
 
-```xml
+* `AnimationDuration` - duration of the popup animation (in milliseconds)
+* `VerticalOffset` - vertical offset of the popup animation
+* `HorizontalOffset` - horizontal offset of the popup animation
 
-<controls:InAppNotification RenderTransformOrigin="0.5,1" />
+## Styling
 
-```
+### Using styles
+
+The In App Notification control is designed to support multiple styles. 
```

Comment:
Should this be "InAppNotification control"?

Reply:
"in-app notification" is better.